src/Core/Vipr.Core/CodeModel/Vocabularies/Capabilities/OdcmInsertCapability.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Vipr.Core.CodeModel.Vocabularies.Capabilities
+{
+    public class OdcmInsertCapability : OdcmBooleanCapability
+    {
+        public override string TermName
+        {
+            get { return "Org.OData.Capabilities.V1.InsertRestrictions"; }
+        }
+
+        /// <summary>
+        /// Entities can be inserted
+        /// </summary>
+        public bool Insertable
+        {
+            get { return BooleanValue; }
+        }
+
+        public OdcmInsertCapability()
+        {
+            BooleanValue = true;
```

Comment:
These constructors capture the default value of an annotation as specified in the [capabilities edmx] (http://docs.oasis-open.org/odata/odata/v4.0/os/vocabularies/Org.OData.Capabilities.V1.xml).
It is just a coincidence that all these capabilities support a default value of true.